src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -160,14 +160,61 @@ class ActivityFeed extends React.Component<Props, State> {
         versionInfoHandler(data);
     };
 
+    /**
+     * Determine whether or not a sort should occur, based on new comments, tasks, versions.
+     *
+     * @param {Comments} - [comments] - Object containing comments for the file.
+     * @param {Tasks} - [tasks] - Object containing tasks for the file.
+     * @param {FileVersions} - [versions] Object containing versions of the file.
+     * @return {boolean} True if the feed should be sorted with new items.
+     */
+    shouldSortFeedItems(comments?: Comments, tasks?: Tasks, versions?: FileVersions): boolean {
+        if (!comments || !tasks || !versions) {
```

Comment:
Why not just return what's in the if() &ed

Reply:
We decided to just do a single sort instead of merging and sorting as data was coming in.

Reply:
I think his point was why not ```return (!comments || !tasks || !versions)```?

Reply:
I have a feeling that allergy meds kicked in around this time. Done.